core/trino-main/src/main/java/io/trino/operator/AggregationOperator.java

```diff
@@ -176,4 +169,19 @@ public Page getOutput()
         state = State.FINISHED;
         return pageBuilder.build();
     }
+
+    private boolean updateMemory()
+    {
+        long memorySize = 0;
+        for (Aggregator aggregate : aggregates) {
+            memorySize += aggregate.getEstimatedSize();
+        }
+        if (useSystemMemory) {
+            systemMemoryContext.setBytes(memorySize);
+        }
+        else {
+            userMemoryContext.setBytes(memorySize);
+        }
+        return true;
```

Comment:
add a comment that this method returns boolean to match `UpdateMemory` interface

Reply:
Done